src/roman_datamodels/stnode/_mixins.py

```diff
@@ -13,16 +13,20 @@
 from ._schema import Builder, _get_keyword, _get_properties
 from ._tagged import _get_schema_from_tag
 
+# This is a workaround for MyPy to understand the Mixin classes
 if TYPE_CHECKING:
     from typing import ClassVar, TypeAlias
 
     from astropy.time import Time
 
     from ._tagged import TaggedObjectNode, TaggedScalarNode
 
+    class _TimeNode(Time, TaggedScalarNode):
+        pass
+
```

Comment:
Hacky way to do this but you can't double inherit from the same object nor is mypy happy with a tuple unpack.